tests/integration/plugins/python_v2/test_python.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2025 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import os
+import subprocess
+import sys
+import textwrap
+from pathlib import Path
+
+import pytest
+import yaml
+from craft_parts import LifecycleManager, Step, plugins
+from craft_parts.plugins.python_v2.python_plugin import PythonPlugin
+
+pytestmark = pytest.mark.slow
```

Comment:
When you merge from main you'll need this:

```suggestion
pytestmark = [pytest.mark.slow, pytest.mark.python]
```

Reply:
Sticks the integration tests in the Python group for our newly separated test suites, so it'll run in that group rather than the common group.